dotcom-rendering/src/components/CardPicture.tsx

```diff
@@ -128,9 +126,13 @@ export const CardPicture = ({
 	loading,
 	roundedCorners,
 	isCircular,
-	aspectRatio = '5:3', // Default aspect ratio
```

Comment:
I'm in two minds about if we should keep the default and always request a 5:3 crop unless specified. Any thoughts welcome.